ERROR, 2023-10-31 13:12:06,993, Ошибка division by zero в функции func_2 при аргументах (5, 0), {}
ERROR, 2023-10-31 13:12:06,994, Ошибка 'l' в функции my_func при аргументах ({'f': '1', 'd': '3', 's': '5'}, 'l', 23), {}
